packages/seed/fern/definition/config.yml

```diff
@@ -1,7 +1,15 @@
 imports:
   features: features.yml
 types:
+  MinimalSeedWorkspaceConfiguration:
```

Comment:
This is to allow the CLI to have it's own seed folder and `seed.yml` without any of the other metadata we currently track for seed

Reply:
Allowing us to add in additional configuration down the line, as well as specify a location for the changelog within another project

Reply:
Up to call this `CliSeedWorkspaceConfiguration` ? dont foresee us needing to add more so down to make the naming here hyperspecific